src/Cake.Svn/Add/SvnAddSettings.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace Cake.Svn.Add
+{
+    /// <summary>
+    /// Settings for <see cref="SvnAdder"/>.
+    /// </summary>
+    public sealed class SvnAddSettings : SvnSettings
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SvnAddSettings"/> class.
+        /// </summary>
+        public SvnAddSettings()
+        {
+            Depth = SvnDepth.Infinity;
+            AddParents = true;
+        }
+
+        /// <summary>
+        /// If TRUE, recurse up path's directory and look for a versioned directory. If found,
+        /// add all intermediate paths between it and path. If not found, fail with 
+        /// SVN_ERR_CLIENT_NO_VERSIONED_PARENTS.
+        /// </summary>
+        public bool AddParents { get; set; }
+
+        /// <summary>
+        /// If force is not set and path is already under version control, return the error
+        /// SVN_ERR_ENTRY_EXISTS. If force is set, do not error on already-versioned items.
```

Comment:
`SVN_ERR_ENTRY_EXISTS` is a SharpSvn constant. IMHO they are not useful for users of the addin.